### PR TITLE
Minor Rev changes

### PR DESCRIPTION
**Pull request by coolmankid12345**
## About the PR

Lowered cost of Mindshield crate to 3000, changed some text, and changed mindshield to mapinit instead of componentinit.
## Why / Balance

Mindshields might be way too expensive right now so I'm going to lower the cost and see how that goes but I'm open to changes on the price.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Content.Shared/Revolutionary/SharedRevolutionarySystem.cs [hunk 1]
```diff
@@ -14,13 +14,13 @@ public sealed class SharedRevolutionarySystem : EntitySystem
     public override void Initialize()
     {
         base.Initialize();
-        SubscribeLocalEvent<MindShieldComponent, ComponentInit>(MindShieldImplanted);
+        SubscribeLocalEvent<MindShieldComponent, MapInitEvent>(MindShieldImplanted);
```
Comment (Partmedia) [inline Content.Shared/Revolutionary/SharedRevolutionarySystem.cs:17]: I didn't see the reason for this in the PR description, could you summarize why it should be `MapInitEvent`?

Comment (moonheart08) [reply Content.Shared/Revolutionary/SharedRevolutionarySystem.cs:17]: ComponentInit is way too early in the initialize process. This is correct.
